app/src/main/java/com/cpr3663/cpr_scouting_app/Match.java

```diff
@@ -204,6 +208,8 @@ public void onClick(View view) {
                         start_Match();
                         break;
                     case Constants.PHASE_AUTO:
+                        // If we're going to teleop manually, log the start time offset
+                        Globals.EventLogger.LogData(Constants.LOGKEY_START_TIME_OFFSET, String.valueOf(Math.round((TIMER_AUTO_LENGTH * 1000.0 - System.currentTimeMillis() + startTime) / 10.0) / 100.0));
```

Comment:
nope.  That's just 15secs "ago".  I need to know "how late did you hit Start".  So if you jump to teleop at 13secs, i need to log the offset of 2secs (because we already filed a bunch? of auto events with the wrong time and we can't fix it here).  We can fix it in Tableau Prep if we know the offset to use.

Reply:
Oh, I thought you were getting the start time not the offset. The order was confusing. But if you reorder it you are start - current to get the negative elapsed time then adding the 15, to get a negative number which is the offset.